BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1696,6 +1689,11 @@ IEnumerator FlightAnimRoutine()
         }
         IEnumerator BoostRoutine()
         {
+            if (weaponClass == WeaponClasses.SLW && FlightGlobals.getAltitudeAtPos(part.transform.position) > 0)
+            {
+                yield return new WaitWhileFixed(() => vessel.Splashed); //don't start torpedo thrust until underwater
```

Comment:
I see `dropTime` being used in a number of other places, particularly as `TimeIndex - dropTime`. We probably want to set `dropTime = TimeIndex;` for torpedoes once the torp splashes down.

Reply:
yes, there's a couple places in the guidance section where things start immediately after dropTime is up, so adjusting dT should be done. As should probably setting it to a high temporary placeholder for the duration of the drop.